suites/selftest/ts/package.xml

```diff
@@ -5,6 +5,30 @@
     <description>TE Self Tests</description>
     <author mailto="te-maint@oktetlabs.ru"/>
 
+    <script-template id="default_arg_test" name="default_args">
```

Comment:
Thinking more about it I've come to conclusion that is it not a script template. It is rather a run template.
Script in tester has no args specification. It has parameters, but not args (parameter+value).
Moreover, the same arguments defaults could be applicable to a session or a package.
So, it should be run-template with a name attribute (template name, not "id" used here).
If we really want to specify script in a template, we should do it inside similar to 'run' specification.
Basically it makes script optional in run specification.

For me it is still unclear what is the best way to refer to template in this case. Initially I thought that it should be done using run attribute like template="template-name", but may be it is better to add template as an element to make it an alternative to script/session/package. However, it should make script/session/package specification mandatory in a template, but it is not always desirable.

@ol-arteman What do you think?

Reply:
Since there is no other ideas and thoughts, please, do as I suggest above.

Let's use an attribute to refer to run template from run element.